ui/org.eclipse.pde/META-INF/MANIFEST.MF

```diff
@@ -2,6 +2,6 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.pde;singleton:=true
-Bundle-Version: 3.13.2600.qualifier
+Bundle-Version: 3.13.2601.qualifier
```

Comment:
Is this version-bump necessary?

Reply:
I thought  I saw an error for this, but couldn't see it now! will revert.

Reply:
@HannesWell - done, thanks!